main.py

```diff
@@ -2,6 +2,55 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # 1. Create:
+    genres_for_create = [
+        "Western",
+        "Action",
+        "Dramma"
+    ]
```

Comment:
The variable `genres_for_create` is a list of strings, which is not recommended according to the task checklist. Consider changing it to a list of tuples for better iterating.